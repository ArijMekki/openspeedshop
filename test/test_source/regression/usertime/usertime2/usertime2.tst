echo expcreate -f ../../../../executables/mutatee/cplus_version/mutatee usertime >> input.script
echo expgo >> input.script
echo expview stats -m usertime::inclusive_time >>input.script
openss -batch < input.script
