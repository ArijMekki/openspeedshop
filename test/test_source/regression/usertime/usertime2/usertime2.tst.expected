      Inclusive Time  Function Name
              6.5000  __libc_start_main(libc.so.6.1)
              6.5000  work(mutatee: mutatee.c,33)
              6.5000  _start(mutatee)
              3.0000  f3(mutatee: mutatee.c,27)
              2.0000  f2(mutatee: mutatee.c,18)
              1.0000  f1(mutatee: mutatee.c,9)
